merkle-tree/indexed/src/lib.rs

```diff
@@ -80,7 +82,7 @@ where
             }
             // The value of `new_element` needs to be lower than the value of
             // next element pointed by `old_low_element`.
-            if new_element.value >= low_element.next_value {
+            if new_element.value >= low_element_next_value {
```

Comment:
Is the output of all the hash functions we have a trait for consistently be?

Reply:
Done with `ark_ff::BigInteger`/`ark_ff::BigInt`.

And from now on, I'm going to avoid `num_bigint`, because they use vectors:

https://docs.rs/num-bigint/0.4.4/src/num_bigint/biguint.rs.html#38-40